Polyhedron/demo/Polyhedron/Plugins/PMP/Join_and_split_polyhedra_plugin.cpp

```diff
@@ -179,7 +167,7 @@ void Polyhedron_demo_join_and_split_polyhedra_plugin::on_actionSplitPolyhedra_tr
         compute_color_map(item->color(), new_polyhedra.size(), std::back_inserter(color_map));
       Scene_group_item *group = new Scene_group_item("CC");
        scene->addItem(group);
-      for(FaceGraph* polyhedron_ptr : new_polyhedra)
+      for(FaceGraph polyhedron_ptr : new_polyhedra)
```

Comment:
You're copying the complete graph currently; the constructor of `Scene_surface_mesh_item` also looks like it's copying the graph another time afterwards.

If it's not a pointer anymore, it shouldn't be called `_ptr`.